### DISCUSSION TITLE
Is there some way to switch node version only for current terminal?

### DISCUSSION THREAD

**Discussion by idlist** [coreybutler/nvm-windows#1018] [Support] [answered]
e.g., I want to use Node 20 most of the time, but need to use Node 12 for some specific reasons for some short time, and it would be better to only set Node version for this terminal so that I would not need to switch back and forth 🤔

**Comment by coreybutler** [comment, marked as answer]
There is no native way to do this. It comes down to the difference of using symlinks over shims. There will be a way to do this in [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime).

Even though there is no native way to do this, there is a workaround. The node executable can be called directly. See https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#how-can-i-use-two-different-versions-of-nodejs-at-the-same-time. You can find the Node installation root by running `nvm debug`.
